### DISCUSSION TITLE
Liquid Cooling - Coolant Temps

### DISCUSSION THREAD

**Discussion by kyk85** [mathoudebine/turing-smart-screen-python#379] [Q&A]
Did some searching and couldn't find any information regarding this so I thought I'd ask here.

Does this or the original Turing Smart Screen 5" software support detection of coolant temps? In my case it would be the Asus Z370-E's T-sensor.

**Comment by mathoudebine** [comment]
Hello! I can't say for the original software, for this program it is not supported for now.
It could be added if it is supported by LibreHardwareMonitor library:

Can you download the latest version here:
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/releases/download/v0.9.2/LibreHardwareMonitor-net472.zip
Then run LibreHardwareMonitor.exe and tell me if you see your coolant temps in the list?

**Reply by Jayhovahv** [reply]
Hello Matthieu - I am not the OP but I am looking to do the same thing. I have a T_Sensor on my Asus B550-F Motherboard and it shows up as Temperature#5 (See screenshot) in Libre Hardware Monitor 0.9.3. Would displaying this be supported or could it be added? Thank you!
![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/66964783/2b57b037-acb4-4cfc-a3c8-95ee23b5b782)
